roles/pulp_content_guard/README.md

```diff
@@ -10,9 +10,16 @@ Role variables
 * `pulp_username`: Username used to access Pulp server. Default is `admin`
 * `pulp_password`: Password used to access Pulp server. Default is unset
 * `pulp_validate_certs`: Whether to validate Pulp server certificate. Default is `true`
-* `pulp_content_guard_x509_cert_guards`: List of x509 cert guards. Each item is
-  a dict with the following keys: `name`, `description`, `ca_certificate`,
-  `state`.
+* `pulp_content_guard_x509_cert_guards`: List of x509 cert guards to create/update/delete. Each item is
+  a dict containing: 
+  * `name` (Required)
+  * `description`
+  * `ca_certificate`
+  * `state` (default is `present`. Setting this value to `absent` will delete the content guard if it exists)
+* `pulp_content_guard_rbac`: List of groups to create/update/delete. Default is an empty list. Each item is a dict containing:
```

Comment:
`s/groups/RBAC content guards/`

Reply:
I'm not sure what you mean by this, would you mind explaining further?

Reply:
It's a `sed` expression - find `groups`, replace with `RBAC content guards`

Reply:
Ah, I see it now. Thanks!